### DISCUSSION TITLE
CP2K farm MPI HTCONDOR

### DISCUSSION THREAD

**Discussion by samothyear** [cp2k/cp2k#2357] [General]
Dear readers and developers,

I read about CP2K farm with MPI and SLURM but my question is: 
has anyone (or is possible to) run a CP2K farm using MPI and HtCondor? 
Not sure if this is appropriate here.

Thanks in advance for your time and your answers.

Sincerely,

Sam.

**Comment by alazzaro** [comment]
I'm not sure I understand the question... 
I think you are asking if you can run an MPI application (like CP2K) on HtCondor? If so, then it seems [it is possible](https://htcondor.readthedocs.io/en/latest/users-manual/parallel-applications.html). Again, this is not CP2K related...

**Reply by samothyear** [reply]
Dear Alfio Lazzaro,

thank you for your answer but I was looking for the opinion of someone with direct experience of using cp2k in an hpc infrastructure which uses htcondor as scheduler.

Sincerely
Sam.

**Reply by alazzaro** [reply]
Then I suggest you to post your question on the google forum (https://groups.google.com/g/cp2k/) where you can reach other users. This is really for CP2K developments...

**Reply by samothyear** [reply]
Ok Alfio,

thank you for your hint.
I'll try there.

Sincerely.
